multi-arch-manifest.yaml

```diff
@@ -1,4 +1,5 @@
-image: subspacecommunity/subspace
+image: subspacecommunity/subspace:latest
+tags: ['1.1.1', '1.1', '1']
```

Comment:
So when we update to `2` we then have to change this to `[2.CURRENT_MINOR_VERSION.CURRENT_BUG_FIX_VERSION, 2.CURRENT_MINOR_VERSION, 2]` ?

I assume that we won't ever build off of master in dockerhub as we only want it to trigger when we release a version?

Reply:
Hmmmm true, we can avoid pushing incomplete versions as images in the master branch